resources/integration_test/tests/queries.yaml

```diff
@@ -80,21 +80,21 @@ tests:
     test_definition: SELECT enrolled AS e, studentid AS s, name AS n FROM class
     db: integration_test
     expected_result:
-      - [ true, John, 10329 ]
-      - [ false, Jane, 342 ]
-      - [ true, Mike, 303 ]
-      - [ false, Mary, 204323 ]
-      - [ false, Pete, 10 ]
-    expected_bson_type: ["bool", "string", "int"]
+      - [ true, 10329, John ]
+      - [ false, 342, Jane  ]
+      - [ true, 303, Mike ]
+      - [ false, 204323, Mary ]
+      - [ false, 10, Pete ]
+    expected_bson_type: ["bool", "int", "string"]
```

Comment:
The test updates look correct, but the result-set test is failing because it is expecting an `int` column but got a `string`.
Not sure why, I tried running the test generator to confirm the values but that's hitting a different error entirely when building. 

```
[2023/11/13 20:20:24.522]    --> integration_test\src\test_runner\test_generator_util.rs:159:53
[2023/11/13 20:20:24.522] 159 |                 SqlDataType::EXT_GUID => CDataType::Guid,
[2023/11/13 20:20:24.522]     |                                                     ^^^^ variant or associated item not found in `CDataType`
```

Reply:
this test is expected to fail still until mongosql gets released, and then subsequent changes are made in monoghouse. So for now, I have been testing with a local fork of both, which shows these tests passing

Reply:
Ah gotcha.  Thanks for explaining 👍